<launch>
  <node  name="joy_node"     pkg="joy"     type="joy_node"      output="screen">
  </node>
  <node  name="car_control_node"     pkg="canopen"     type="car_control"      output="screen"  launch-prefix="gnome-terminal -e">
  </node>
   <node  name="auto_run_node"     pkg="canopen"     type="auto_run"      output="screen"  launch-prefix="gnome-terminal -e">
  </node>

</launch>
